File: week-3/WD1/fizzbuzz.py
i = 0

while i <= 100:

    if (i % 3 == 0) and (i % 5 == 0) or ("3" in str(i) and "5" in str(i)) or (i % 3 == 0 and "5" in str(i)) or (i % 5 == 0 and "3" in str(i)):
        print ('fizzbuzz')

    elif i % 3 == 0:
        print ('fizz')

    elif i % 5 == 0:
        print ('buzz')


    elif "3" in str(i):
        print ('fizz')

    elif "5" in str(i):
        print ('buzz')

    else:
        print (i)
    i += 1

File: week-7/WD2/fizzbuzz.js
'use strict';

for ( var a = 0; a <= 100; a++) {
  if (a % 15 === 0) {
    console.log('fizzbuzz');
  } else if (a % 3 === 0) {
    console.log('fizz');
  } else if (a % 5 === 0) {
    console.log('buzz');
  } else {
    console.log(a);
  }
}
